torch/ao/quantization/fx/_model_report/model_report.py

```diff
@@ -461,7 +487,11 @@ def generate_qconfig_mapping(self) -> QConfigMapping:
         These configs are based on the suggestions provided by the ModelReport API
         and can only be generated once the reports have been generated.
 
-        Returns a QConfigMapping for the quantization configuration
+        Args:
+            update_qconfig_info_function (Callable) takes in a function that takes in two DetectorQConfigInfo
+            and updates the one that is being compiled
+
+        Returns a Dict mapping module_fqns to DetectorQConfigInfo objects
```

Comment:
this does not look like a qconfig_mapping, should we change the name?

Reply:
changed it to `_generate_module_fqn_to_detector_info_mapping`